testgrid/specs/deploy.yaml

```diff
@@ -56,11 +56,9 @@
     kubernetes:
       version: 1.19.x
     docker:
-      version: 19.03.x
+      version: 20.10.x
```

Comment:
add one more for k8s123-minimal with docker 20.10.x

Reply:
you shouldn't have removed the unsupportedOSIDs for Docker 19.03 though
that'll still fail on ubuntu

Reply:
what the heck github, I saw this as a whole new test leaving the 19.03 one alone

Reply:
looks good

Reply:
oh, nope was just looking at commit view before
yes, you should definitely keep unsupportedOSIDs for the 19.03 test